asammdf/gui/widgets/file.py

```diff
@@ -465,7 +465,7 @@ def __init__(
 
                 item = QtWidgets.QTreeWidgetItem()
                 item.setText(0, "Raw size")
-                item.setText(1, f"{size / 1024 / 1024:.1f} MB")
+                    item.setText(1, f"{size / 1024 / 1024:.1f} MB")
```

Comment:
I believe that this indentation is wrong here

Reply:
Indeed, its a remainder from a local change I made for the channel size as discussed in #645 , my bad.